README.md

```diff
@@ -32,34 +33,51 @@ Explanations for commands.
 - `nvm use` : Install and use the correct version of Node.
 - `npm i` : As stated above; Install dependencies and link local packages.
 
+Related files.
+- `.nvmrc` : Defines the node version used to compile the theme.
+- `package.json and package-lock.json` : Defines the node modules for compiling the theme.
```

Comment:
> - `package.json and package-lock.json` : Defines the node modules for compiling the theme.

Suggestion: 
- `package.json and package-lock.json` : Defines the node modules and scripts for compiling the theme.

Reply:
Fixed! 🦖